src/test/java/com/amazonaws/encryptionsdk/TestUtils.java

```diff
@@ -174,4 +174,39 @@ public static ByteArrayInputStream insecureRandomStream(int length) {
       };
       return frameSizeToTest;
   }
+
+    /**
+     * Converts an array of unsigned bytes (represented as int values between 0 and 255 inclusive)
+     * to an array of Java primitive type byte, which are by definition signed.
+     * @param unsignedBytes An array on unsigned bytes
+     * @return An array of signed bytes
+     */
+  public static byte[] unsignedBytesToSignedBytes(final int[] unsignedBytes) {
+      byte[] signedBytes = new byte[unsignedBytes.length];
+
+      for(int i= 0 ; i < unsignedBytes.length; i++) {
```

Comment:
Here and throughout: space after `if`, `for`, and other keywords to distinguish them from function calls.

Here: Space on either side of the equals.

Reply:
Fixed